CHANGES.rst

```diff
@@ -1,21 +1,22 @@
 Changes
 =======
 
-Version 0.9.0 (UNRELEASED)
+Version 0.9.0 (2022-12-13)
```

Comment:
We should also add an entry for the workspace utils ([workspace: move from reana-workflow-controller](https://github.com/reanahub/reana-commons/commit/9cec0269e4cdfb1657019734edc42ba123770734) and following commits) and one entry to summarize all the changes we have made to r-server's OpenAPI (e.g. improved specification of the responses)